context/app/static/js/components/Contexts.tsx

```diff
@@ -10,6 +10,8 @@ export interface FlaskDataContextType {
   title: string; // preview page title
   vis_lifted_uuid?: string;
   redirected?: boolean;
+  redirectedFromId?: string;
+  redirectedFromPipeline?: string;
```

Comment:
The `.get` without a second argument will resolve to `None` if the key isn't present and will then be `null` once converted to json. The types here should be `?: string | null`.

Reply:
Got it - is this preferable to providing a false second value to `.get`?